packages/connect/src/connect.js

```diff
@@ -143,47 +196,43 @@ export function connect(Comp, options) {
         );
       }
 
-      // add a custom deriveStoresFromProps lifecyle method
-      static getDerivedStateFromProps(props, state) {
-        if (super.deriveStoresFromProps) {
-          // inject all local stores and let the user mutate them directly
-          const stores = mapStateToStores(state);
-          super.deriveStoresFromProps(props, ...stores);
-        }
-        // respect user defined getDerivedStateFromProps
-        if (super.getDerivedStateFromProps) {
-          return super.getDerivedStateFromProps(props, state);
-        }
-        return null;
-      }
-
```

Comment:
What was the purpose of this custom `getDerivedStateFromProps` and why was it removed now?

I see that enables us to also remove the `mapStateToStores` (at the top of the file) but I don't understand the purpose.

Reply:
It is a new lifecycle that `react-easy-state` introduced to be able to manipulate stores that are local to a class component. I don't think it is relevant to our use case.

Reply:
okay, I see. thanks!